assets/css/admin.css

```diff
@@ -0,0 +1,161 @@
+.patreon-options-info {
+	max-width : 800px;
+	width : 100%;
+	display:block;
+}
+
+#patreon_attachment_patreon_level_form{
+	display:inline-block;
+	width : 100%;
+	text-align:left;
+	margin: 0px;
+	font-weight: bold;
+}
+#patreon_attachment_patreon_level{
+	
+	width : auto;
+	padding-left: 3px;
+}
+
+.patreon_image_locking_interface_heading{
+	
+	font-size :150%;
+	font-weight : bold;
+	margin-left: 15px;
+	display : inline-block;
+}
+
+.thickbox-loading{
+	max-width : 500px !important;
+	width : 100% !important;
+	min-height : 265px !important;
+	
+	
+}
+.patreon_image_locking_interface_content{
+	background-color : #f3f3f3;
+	width :100%;
+	display:inline-table;
+	border-top : 2px solid #dedede;
+	border-bottom : 2px solid #dedede;
+	padding: 15px;
+	-webkit-box-sizing: border-box; /* Safari/Chrome, other WebKit */
+	-moz-box-sizing: border-box;    /* Firefox, other Gecko */
+	box-sizing: border-box;         /* Opera/IE 8+ */
+	color : #737276;
+	font-weight:bold;
+	min-height:150px;
+	
+}
+.patreon_image_locking_interface_input_prefix
+{
+    background-color: #FFF !important;
+    line-height:20px;
+    height:26px;
+    padding:2px;
+    font-size:14px;
+	color : #000000;

```

Comment:
why the extra indent?

Reply:
This indent doesnt exist when viewed in notepad++  - im guessing some display issue at github? If it doesnt display the same when downloaded that is...